src/Rules/Generators/GhcSplit.hs

```diff
@@ -0,0 +1,25 @@
+module Rules.Generators.GhcSplit (generateGhcSplit) where
+
+import Base
+import Expression
+import Oracles
+import Settings.User
+
+generateGhcSplit :: Expr String
+generateGhcSplit = do
+    let yesNo = lift . fmap (\x -> if x then "YES" else "NO")
+    perl <- getBuilderPath Perl
+    let script = "driver" -/- "split" -/- "ghc-split.prl"
```

Comment:
Good to know. Was wondering about the use of `"x/y/z"` and `"x" -/- "y" -/- "z"`.